app/tests/unit/lib/helpers/reportStatusHelpers.test.ts

```diff
@@ -0,0 +1,184 @@
+import { screen } from "@testing-library/react";
```

Comment:
I'm testing the helpers like this because I want to make sure they work together properly and show the right status based on the given data. I'm not convinced this test is the best way to do that, though--thoughts?

Reply:
It sounds overkill to use testing-library and graphql to test some simple functions, no?